manifests/s/Squitch/Tess/0.6.0/Squitch.Tess.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Squitch.Tess
+PackageVersion: 0.6.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Commands:
+- tess
+- tess-cli
+Installers:
+- Architecture: x86

```

`Exception trying to launch executable C:\Program Files\Tess\tess-cli.exe: System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)`

```suggestion
- Architecture: x64
```